File: Server/aoRss/Models/Db/RSSAggregatorSourceRuleModel.vb

Namespace Models.Db
    Public Class RSSAggregatorSourceRuleModel
        Inherits BaseModel
        Implements ICloneable
        '
        '====================================================================================================
        '-- const
        Public Const contentName As String = "RSS Aggregator Source Rules"                   '<------ set content name
        Public Const contentTableName As String = "aoRSSAggregatorSourceRules"            '<------ set to tablename for the primary content (used for cache names)
        Private Shadows Const contentDataSource As String = "default"   '<------ set to datasource if not default
        '
        '====================================================================================================
        ' -- instance properties
        Public Property AggregatorID As Integer
        Public Property SourceID As Integer
        Public Property link As String
        Public Property articleName As String
        Public Property pubdate As String
        Public Property description As String
    End Class
End Namespace

File: Server/aoRss/Models/Db/RSSAggregatorSourceRuleModel.cs
using Contensive.Models.Db;
using System;

namespace Contensive.Addons.Rss.Models.Db {
    public class RSSAggregatorSourceRuleModel : DbBaseModel {
        //
        //====================================================================================================
        /// <summary>
        /// table definition
        /// </summary>
        public static DbBaseTableMetadataModel tableMetadata { get; } = new DbBaseTableMetadataModel("RSS Aggregator Source Rules", "aoRSSAggregatorSourceRules", "default", false);
        // 
        // ====================================================================================================
        // -- instance properties
        public int AggregatorID { get; set; }
        public int SourceID { get; set; }
        public string link { get; set; }
        public string articleName { get; set; }
        public string pubdate { get; set; }
        public string description { get; set; }
    }
}